.travis.yml

```diff
@@ -30,6 +17,27 @@ before_install:
   - make install
   - cd ..
 
+script:
```

Comment:
Can't we paralellize that? https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix